core/src/main/scala/org/scalarelational/column/property/NotNull.scala

```diff
@@ -1,8 +0,0 @@
-package org.scalarelational.column.property
-
-/**
- * @author Matt Hicks <matt@outr.com>
- */
-object NotNull extends ColumnProperty {
```

Comment:
I would recommend adding a spec to truly understand the problem here.

Write a query in ScalaRelational filtering Optional is None and another query filtering Optional is Some. In SQL this would look like:

```sql
SELECT * FROM table WHERE table.optional IS NOT NULL
``` 

How are you going to represent that?  Add this to a spec and I think you'll understand the problem.